frontend/src/pages/onboarding/index.tsx

```diff
@@ -119,7 +131,7 @@ export default function Onboarding() {
   const { handleSubmit, trigger, watch } = methods;
 
   // for debugging
-  useWatchForm(watch);
+  useDevWatchForm(watch);
 
```

Comment:
The bouldering/top rope/lead climbing/SCNS grades are optional. Hence, the hint text should have (optional) e.g. Grade (optional). Also, it seems that FE sends some invalid data to BE when the blank option is selected (See the ClimbJios Staging Alerts tele channel)